openslides_backend/action/actions/committee/import_meeting.py

```diff
@@ -0,0 +1,264 @@
+import time
```

Comment:
The meeting was renamed by @tsiegleauq to `meeting.import` (see the wiki for a different payload!). Please move this file and the test into the meeting realm

Reply:
Renamed.